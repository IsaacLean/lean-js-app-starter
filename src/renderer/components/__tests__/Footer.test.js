import Adapter from 'enzyme-adapter-react-16'
import Enzyme, { shallow } from 'enzyme'
import React from 'react'

import Footer from '../Footer'

Enzyme.configure({ adapter: new Adapter() })

describe('Footer', () => {
  test('snapshot', () => {
    const component = shallow(<Footer />)
    expect(component).toMatchSnapshot()
  })
})
