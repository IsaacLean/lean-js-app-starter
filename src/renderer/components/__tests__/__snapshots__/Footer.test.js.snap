// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "Show:",
        " ",
        <FilterLink
          filter="all"
        >
          All
        </FilterLink>,
        " ",
        <FilterLink
          filter="active"
        >
          Active
        </FilterLink>,
        " ",
        <FilterLink
          filter="completed"
        >
          Completed
        </FilterLink>,
      ],
    },
    "ref": null,
    "rendered": Array [
      "Show:",
      " ",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "All",
          "filter": "all",
        },
        "ref": null,
        "rendered": "All",
        "type": [Function],
      },
      " ",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Active",
          "filter": "active",
        },
        "ref": null,
        "rendered": "Active",
        "type": [Function],
      },
      " ",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Completed",
          "filter": "completed",
        },
        "ref": null,
        "rendered": "Completed",
        "type": [Function],
      },
    ],
    "type": "p",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "Show:",
          " ",
          <FilterLink
            filter="all"
          >
            All
          </FilterLink>,
          " ",
          <FilterLink
            filter="active"
          >
            Active
          </FilterLink>,
          " ",
          <FilterLink
            filter="completed"
          >
            Completed
          </FilterLink>,
        ],
      },
      "ref": null,
      "rendered": Array [
        "Show:",
        " ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "All",
            "filter": "all",
          },
          "ref": null,
          "rendered": "All",
          "type": [Function],
        },
        " ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Active",
            "filter": "active",
          },
          "ref": null,
          "rendered": "Active",
          "type": [Function],
        },
        " ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Completed",
            "filter": "completed",
          },
          "ref": null,
          "rendered": "Completed",
          "type": [Function],
        },
      ],
      "type": "p",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
